spec/models/location_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Location, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Are there any tests you think would be valuable to add here? if not, should we just delete this file until we do?

Reply:
Good catch, thanks Ben!